### PR TITLE
Updated UI to prepare to allow classifications

### PR DESCRIPTION
**Pull request by shollingsworth**
* adjusted dialog for lower parts of the image
* updated for mobile
* Added display div, added scores, adjusted the way labels are applied.
* Classification widgets

### CODE REVIEW

File: frontend/src/lib/util.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    }, time)
+    this.start = Date.now()
+}
+
+export function Timer(callback: Function, time: number) {
+    // @ts-ignore
+    this.setTimeout(callback, time)
+}
+
+Timer.prototype.setTimeout = function (callback: Function, time: number) {
```
Comment (feydan) [inline frontend/src/lib/util.ts:110]: This looks like a duplicate (declared above as well)

**Review by feydan — CHANGES_REQUESTED** [review]
One small comment -- looked like there is a duplicate prototype function declaration in util.ts

**Comment by shollingsworth** [comment]
@feydan changes are made, also removed a duplicate about.vue file.

**Review by feydan — APPROVED** [review]
looks good!

**Pull request merged**